dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/ContentletAPIInterceptor.java

```diff
@@ -1863,6 +1863,30 @@ public Contentlet copyContentlet(Contentlet contentlet, Folder folder, User user
 		return c;
 	}
 
+	@Override
+	public Contentlet copyContentlet(final Contentlet contentletToCopy,
+									 final Host host, final Folder folder, final User user, final String copySuffix,
+									 final boolean respectFrontendRoles) throws DotDataException, DotSecurityException, DotContentletStateException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotmarketing.portlets.contentlet.business.DotContentletStateException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)